CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
+## 0.3.1
+
+- Added `SharedStdIn.nextLine` (similar to `readLineSync`) and `lines`:
+
+```dart
+main() async {
+  // Prints the first line entered on stdin.
+  print(await sharedStdIn.nextLine());
+  
+  // Prints all remaining lines.
+  await for (final line in sharedStdIn.lines) {
+    print(line);
+  }
+}
+```
```

Comment:
Nit: if you indent this +2 lines, it'll be formatted as part of the previous bullet point rather than its own paragraph.

Reply:
Oh great, thanks! I'll make this change everywhere in the CHANGELOG before releasing.